.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
-node_js: "6"
+os:
+  - linux
+  - osx
+node_js:
+  - "node"
```

Comment:
I know that but I'm not sure that we need to use both checks. I'm using node >7, I don't know you and @jens1o.